_topic_maps/_topic_map.yml

```diff
@@ -3222,13 +3222,13 @@ Topics:
   - Name: Virtual machine networking
     Dir: vm_networking
     Topics:
-    - Name: Using the default pod network with OpenShift Virtualization
+    - Name: Configuring the default pod network with OpenShift Virtualization
```

Comment:
I'm not sure about "configuring". We are not configuring the Pod network, it is already configured and available for us. We are merely connecting to the network

Reply:
+1
@bgaydosrh This is about configuring the VM so that it can connect to the default pod network. You could change the title to "Configuring the virtual machine for the default pod network".

Reply:
Done.